ViewModel/Utilities/ImportFromFDA1Helper.cs

```diff
@@ -57,108 +57,110 @@ private static RatingCurveElement CreateRatingElement(RatingFunction rat)
+                ys = CreateNormalDistributions(rat);
+            }
+            else if (rat.ErrorTypesId == ErrorType.TRIANGULAR)
+            {
+                ys = CreateTriangularDistributions(rat);
+            }
+            else if (rat.ErrorTypesId == ErrorType.UNIFORM)
+            {
+                ys = CreateUniformDistributions(rat);
+            }
+            else if (rat.ErrorTypesId == ErrorType.NONE)
+            {
+                ys = CreateDeterministicDistributions(rat);
             }
-        }
-
-        private static UncertainPairedData CreateRatingPairedDataWithPointUncertainty(RatingFunction rat)
-        {
-            UncertainPairedData upd = new UncertainPairedData();
-            //if (rat.ErrorTypesId == ErrorType.LOGNORMAL)
-            //{
-            //    rat._st
-            //    List<Deterministic> yVals = new List<Deterministic>();
-            //    foreach (double d in ys)
-            //    {
-            //        yVals.Add(new Deterministic(d));
-            //    }
-            //    UncertainPairedData curve = new UncertainPairedData(xs.ToArray(), yVals.ToArray(), xLabel, yLabel, name, "", -1);
-            //    return curve;
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.NORMAL)
-            //{
-
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.TRIANGULAR)
-            //{
-
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.UNIFORM)
-            //{
-
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.NONE)
-            //{
 
-            //}
-            return upd;
+            return new UncertainPairedData(rat.GetStage(), ys.ToArray(), "Stage", "Flow", "Rating", "", -1);
         }
 
-        private static UncertainPairedData CreateRatingPairedDataWithGlobalUncertainty(RatingFunction rat)
+        private static List<IDistribution> CreateLogNormalDistributions(RatingFunction rat)
         {
-            UncertainPairedData upd = new UncertainPairedData();
-            string xLabel = "Stage";
-            string yLabel = "Flow";
-            string name = "Rating";
-
-            double[] stages = rat.GetStage();
-            double[] flows = rat.GetDischarge();
-
-            if (rat.ErrorTypesId == ErrorType.LOGNORMAL)
+            List<IDistribution> ys = new List<IDistribution>();
+            for (int i = 0; i < rat.NumberOfPoints; i++)
             {
-                List<IDistribution> ys = new List<IDistribution>();
-                for (int i = 0; i < rat.NumberOfPoints; i++)
+                if (rat.UsesGlobalError)
+                {
+                    ys.Add(new LogNormal(rat.GetDischarge()[i], rat.GlobalStdDevLog));
+                }
+                else
                 {
-                    ys.Add(new LogNormal(flows[i], rat.GlobalStdDev));
+                    ys.Add(new LogNormal(rat.GetDischarge()[i], rat.IndividualLogStDevs[i]));
```

Comment:
Any way we can make the property names consistent between GlobalStd's and IndividualStd's?  Globals are consistent with Globals, and Individuals are consistent with Individuals, but they aren't consistent with each other.  It makes it more confusing and inconsistent when you have to use one or the other based on an arbitrary bool property that doesn't refer to Std in the name.

Reply:
Yes, this is all kind of weird but comes from Bob's importer code. He is distinguishing if the uncertainty value applies to the whole curve or if there is uncertainty around each coordinate in the curve.